test/examples_test.go

```diff
@@ -44,68 +64,87 @@ func TestESportsDataFromPandaScore(t *testing.T) {
 		"YOUR_PANDASCORE_API_KEY",
 	}
 
-	NewProjectTest(t, projectDir).
+	NewTestscriptTest(t, projectDir).
 		ExecuteMakeTarget("build").
 		CopyFile("tmp/x.wasm").
 		CopyFile("config.toml").
 		RenderTemplateFileFromEnvWithCleanup("fn-call.json", requiredEnvVars).
-		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
+		Run(filepath.Join(scriptDir, projectName+".txtar"))
 }
 
-func TestESportsDataFromRimble(t *testing.T) {
-	projectName := "esports_data_from_rimble"
-	projectDir := filepath.Join(examplesDir, projectName)
-	requiredEnvVars := []string{
-		"YOUR_RIMBLE_API_KEY",
-	}
-
-	NewProjectTest(t, projectDir).
-		ExecuteMakeTarget("build").
-		CopyFile("tmp/x.wasm").
-		CopyFile("config.toml").
-		RenderTemplateFileFromEnvWithCleanup(
-			"match-winner.json.template",
-			requiredEnvVars,
-		).
-		RenderTemplateFileFromEnvWithCleanup(
-			"team-kill-diff.json.template",
-			requiredEnvVars,
-		).
-		RunScript(filepath.Join(scriptDir, projectName+".txtar"))
-}
+//func TestESportsDataFromRimble(t *testing.T) {
+//	projectName := "esports_data_from_rimble"
+//	projectDir := filepath.Join(examplesDir, projectName)
+//	requiredEnvVars := []string{
+//		"YOUR_RIMBLE_API_KEY",
+//	}
+//
+//	NewTestscriptTest(t, projectDir).
+//		ExecuteMakeTarget("build").
+//		CopyFile("tmp/x.wasm").
+//		CopyFile("config.toml").
+//		RenderTemplateFileFromEnvWithCleanup(
+//			"match-winner.json.template",
+//			requiredEnvVars,
+//		).
+//		RenderTemplateFileFromEnvWithCleanup(
+//			"team-kill-diff.json.template",
+//			requiredEnvVars,
+//		).
+//		Run(filepath.Join(scriptDir, projectName+".txtar"))
+//}
```

Comment:
Not sure why, but Rimble stopped working all of a sudden. Maybe they changed the API? Could you check on your end? I commented out and was going to make a card to investigate this outside of this PR

Reply:
Specifically, I was getting a 204 response which leads me to believe that my key still works but something about the particular api we are calling may have changed

Reply:
Added TODO with card link